lib/matplotlib/widgets.py

```diff
@@ -1072,9 +1068,20 @@ def set_active(self, index):
         if index not in range(len(self.labels)):
             raise ValueError(f'Invalid CheckButton index: {index}')
 
-        l1, l2 = self.lines[index]
-        l1.set_visible(not l1.get_visible())
-        l2.set_visible(not l2.get_visible())
+        cross_facecolors = self._crosses.get_facecolor()
+        cross_facecolors[index] = colors.to_rgba(
+            "black"
+            if colors.same_color(
+                    cross_facecolors[index], colors.to_rgba("none")
```

Comment:
Extra indent

Reply:
That was a weird editor quirk. Removed, thanks.